docs/pages/configuration/pipelines/functions/create_deployments.mdx

```diff
@@ -2,3 +2,45 @@
 title: Function create_deployments
 sidebar_label: create_deployments
 ---
+
+This function creates a new deployment for a pipeline.
+
+## Arguments
+
+### `--skip-deploy`
+
+The deployment will be skipped.
+
+### `--force-deploy`
+
+The deployment will be forced to run.
+
+### `--sequential`
+
+The deployments will be deployed sequentially, one after another.
+
+### `--render`
+
+The rendered manifests will be printed to the console (stdout) instead of being deployed.
+
+## Usage
+
+```
+create_deployments_all
```

Comment:
flags are passed via `--` not through `_`, so `create_deployments --all` would be correct here

Reply:
fixed